mesheryctl/internal/cli/root/pattern/view_test.go

```diff
@@ -46,6 +47,24 @@ func TestPatternView(t *testing.T) {
 			Token:            filepath.Join(fixturesDir, "token.golden"),
 			ExpectError:      false,
 		},
+		{
+			Name:             "View Specific Pattern by Name",
+			Args:             []string{"view", "design"},
+			ExpectedResponse: "view.pattern.output.golden",
+			Fixture:          "view.pattern.api.response.golden",
+			URL:              testContext.BaseURL + "/api/pattern?name=Untitled Design",
+			Token:            filepath.Join(fixturesDir, "token.golden"),
+			ExpectError:      false,
+		},
+		{
+			Name:             "View Non-Existent Pattern by Name",
+			Args:             []string{"view", "design"},
+			ExpectedResponse: "view.pattern.output.golden",
+			Fixture:          "view.pattern.api.response.golden",
+			URL:              testContext.BaseURL + "/api/pattern?name=NonExistentDesign",
```

Comment:
Similar concern here. A file with this name currently doesn't exist, but if it does get created somehow in the future, this test might fail...